frontend/src/lib/components/widgets/TextInput/TextInput.test.tsx

```diff
@@ -285,17 +263,8 @@ describe("TextInput widget", () => {
       target: { value: "TEST" },
     } as React.ChangeEvent<HTMLInputElement>)
 
-    expect(wrapper.state("value")).toBe("TEST")
-    expect(props.widgetMgr.setStringValue).toHaveBeenCalledWith(
-      props.element,
-      "TEST",
-      {
-        fromUi: true,
-      }
-    )
```

Comment:
Why is this removed?

Reply:
In TextInput.tsx and TextArea.tsx, I removed the committing of value and made sure the value is always dirty when it's changed. The reason was because when you're in a form and we commit the value, then the value is not dirty and therefore the Press Enter to submit form text won't be shown.